specifyweb/frontend/js_src/lib/components/FormSliders/Slider.tsx

```diff
@@ -83,7 +83,7 @@ export function Slider({
           />
         </label>
         <span>/</span>
-        <span>{count}</span>
+        <span className="text-center">{count}</span>
```

Comment:
(optional) can you double-check if this is still needed? compare if it looks better with it or without when the total count is < 9 and when total count is >99